index.js

```diff
@@ -20,7 +20,22 @@ async function run() {
             return;
         }
 
-        const title = github.context.payload.pull_request.title;
+        const owner = github.context.payload.pull_request.base.user.login;
+        const repo = github.context.payload.pull_request.base.repo.name;
+
+        const client = new github.GitHub(process.env.GITHUB_TOKEN);
```

Comment:
I think you could use just `github.token` instead. Doing so should remove the need to set it in the env as well.
```suggestion
        const client = new github.GitHub(github.token);
```
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context